README.md

```diff
@@ -76,4 +77,16 @@ By default, installing a package with the `—save` option, npm saves the packag
 npm config set save-exact
 ```
 
+[↑ back to Table of Contents](#table-of-contents)
+
+## Avoid relying on globally installed packages
+
+### Why?
+
+You can't control and should not depend on the user to install a global depency just because of your project/module.
```

Comment:
Try to rephrase into positive reasons.
* Locally installed modules are shared by developers on the same project. (needs better phrasing)
* Locally installed modules are directly accessible in npm scripts.